CMakeLists.txt

```diff
@@ -391,8 +391,15 @@ add_subdirectory(renderdoc)
 if(ENABLE_QRENDERDOC OR ENABLE_PYRENDERDOC)
     # Make sure Python 3 is found
     set(Python_ADDITIONAL_VERSIONS 3.8 3.7 3.6 3.5 3.4)
-    find_package(PythonInterp 3 REQUIRED)
-    find_package(PythonLibs 3 REQUIRED)
+	if(NOT ${CMAKE_VERSION} VERSION_LESS "3.12")
```

Comment:
Whitespace indentation doesn't match here. I'm not sure if I'm globally consistent in CMakeLists.txt since I don't have clang-format, but it should at least match the surrounding code (which is spaces not tabs).